driver-adapters-wasm/pg-cfpages-basic/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "pg-cfpages-basic",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "test": "jest index.test.js"
+  },
+  "devDependencies": {
+    "@jest/globals": "29.7.0",
+    "jest": "28.1.3",
+    "node-fetch": "2.7.0",
+    "prisma": "5.8.0-dev.10",
+    "wrangler": "https://prerelease-registry.devprod.cloudflare.dev/workers-sdk/runs/6711485115/npm-package-wrangler-2541"
```

Comment:
Pre-release version of this PR: https://github.com/cloudflare/workers-sdk/pull/2541
Needed so that `--node-compat` works for `wrangler pages deploy`, is needed for `pg`.

Reply:
No idea honestly.

That you can link your own registry was completely new to me as well! It's pretty cool.

I hope Renovate will just ignore it. And that Cloudflare soon merges that PR.